src/mocks/threeD.ts

```diff
@@ -61,11 +63,35 @@ export function createFakeViewer({
   cache[project] = {
     viewer,
     modelPromise: Promise.resolve(null),
-    revisionPromise: Promise.resolve(null),
     addEvent: addEvent.bind(null, listeners),
     removeEvent: removeEvent.bind(null, listeners),
     domElement,
   };
 
   return cache[project];
 }
+
+export const fakeModel3DViewerClient: API = {
+  // @ts-ignore
+  revisions3D: {
+    // @ts-ignore
+    retrieve: (): Revision3D => revision3D,
```

Comment:
shouldn't the function be async or return a promise?

Reply:
Fixed